Compilation issue

PR #2332 broke compilation on my MAC...

~~~
../source/dofs/dof_handler.cc:1460:17: error: too few template arguments for class template 'DoFHandler'
      if (*i != DoFHandler<2>::invalid_dof_index)
                ^
../include/deal.II/dofs/dof_objects.h:26:27: note: template is declared here
template <int, int> class DoFHandler;
~~~~~~~~~~~~~~~~~~~       ^
1 error generated.
~~~

--- a/include/deal.II/fe/mapping_q1_eulerian.h
+++ b/include/deal.II/fe/mapping_q1_eulerian.h
@@ -20,6 +20,7 @@
 #include <deal.II/base/std_cxx11/array.h>
 #include <deal.II/base/smartpointer.h>
 #include <deal.II/fe/mapping_q1.h>
+#include <deal.II/dofs/dof_handler.h>
 
 DEAL_II_NAMESPACE_OPEN
 

--- a/include/deal.II/numerics/vector_tools.h
+++ b/include/deal.II/numerics/vector_tools.h
@@ -22,6 +22,8 @@
 #include <deal.II/base/quadrature_lib.h>
 #include <deal.II/base/point.h>
 #include <deal.II/dofs/function_map.h>
+#include <deal.II/dofs/dof_handler.h>
+#include <deal.II/hp/dof_handler.h>
 #include <deal.II/hp/mapping_collection.h>
 
 #include <map>
@@ -37,11 +39,9 @@ template <int dim> class QGauss;
 template <typename number> class Vector;
 template <typename number> class FullMatrix;
 template <int dim, int spacedim> class Mapping;
-template <int dim, int spacedim> class DoFHandler;
 template <typename gridtype> class InterGridMap;
 namespace hp
 {
-  template <int dim, int spacedim> class DoFHandler;
   template <int dim> class QCollection;
 }
 class ConstraintMatrix;

